### ISSUE TITLE
[BUG] Icon does not update when packet loss = 100%

### ISSUE THREAD

**Issue by cybershoe** [cybershoe/PingrThingr#81]
**Describe the bug**
When packet loss is 100%, the status text shows latency as --- and loss as 100%, but the text icon doesn't update

**To Reproduce**
1. Enable text display mode
2. Simulate < 100% packet loss 
3. Wait for an icon update
4. Simulate 100% packet loss
5. Observe lack of icon update

**Expected behavior**
Icon should show latency as "---" with max criticality

**Mac OS Version**
26.4.1 Tahoe

**Additional context**
Appears to be an issue in icons::_evaluate_criticality() at line 51, we mistakenly assume that we will either get both numeric values or both None, while latency=None, loss=1.00 is a valid state

**Comment by cybershoe** [comment]
closed in #94
